.maestro/fire_button/dimiss_fire_during_onboarding.yaml

```diff
@@ -21,13 +21,10 @@ tags:
       - tapOn:
           text: "got it"
       - assertVisible:
-            text: ".*browsing activity with the fire button.*"
-      - tapOn:
-          text: "skip"
-      - assertNotVisible: ".*browsing activity with the Fire Button.*"
+          text: ".*browsing activity with the fire button.*"
       - assertVisible:
-          text: ".*Remember: every time you browse with me a creepy ad loses its wings.*"
+          id: daxDialogDismissButton
       - tapOn:
-            id: "com.duckduckgo.mobile.android:id/fireIconImageView"
-      - tapOn: "Clear All Tabs And Data"
-      - assertNotVisible: "You've got this!.*"
\ No newline at end of file
+          id: daxDialogDismissButton
```

Comment:
nit: i believe it’s redundant to `assertVisible` and then `tapOn` the same element since the `tapOn` will fail if it can’t be found anyway. so it’s slightly faster to omit the first check.

I know we’ve got this pairing throughout the tests but i’ve been removing the first step where i’ve been in making changes.